text/0000_colored-input-output.md

```diff
@@ -0,0 +1,68 @@
+- Feature Name: `colored-input-output`
+- Start Date: 2024-02-07
+- RFC PR: [lf-lang/rfcs#0000](https://github.com/lf-lang/rfcs/pull/0000)
+- Tracking Issue(s): [lf-lang/lingua-franca#0000](https://github.com/lf-lang/lingua-franca/issues/0000)
+
+# Abstract
+[abstract]: #abstract
+
+Magnition mentioned that input and output ports are hard to differentiate if they are placed on the same reactor side.
+Additionally, edges become hard to follow if in large diagrams.
+This is currently not solved by the highlighting mechanism since it highlights the edges by making them thick, which results in the same thickness as the busses.
+
+# Motivation
+[motivation]: #motivation
+
+We want to add color to make it easier to work with large LF models to make them easier to recognize ports or edges if many objects are on the screen.
+This is even more important if we trade layout stability for edge crossings.
+
+Moreover, I want to test this process with a smaller RFC and see if it fits.
+I my opinion this is something to be discussed but I intentionally put two different things in here, since I do not want to do an RFC for them both and they are both effect the color scheme.
```

Comment:
```suggestion
I my opinion this is something to be discussed but I intentionally put two different things in here, since I do not want to do an RFC for them both and they are both affect the color scheme.
```